doc/stages/writers.gdal.rst

```diff
@@ -0,0 +1,123 @@
+.. _writers.gdal:
+
+writers.gdal
+================================================================================
+
+The `GDAL`_ writer creates a raster from a point cloud using an interpolation
+algorithm.  Output is produced using GDAL and can therefore use any `driver
+that supports creation of rasters`_.
+
+.. _`GDAL`: http://gdal.org
+.. _`driver that supports creation of rasters`: http://www.gdal.org/formats_list.html
+
+The technique used to create the raster is a simple interpolation where
+each point that falls within a given radius of a raster cell center
+potentially contributes to the raster's value.
+
+.. note::
+
+    If a circle with the provided radius doesn't encompass the entire cell,
+    it is possible that some points will not be considered at all, including
+    those that may be within the bounds of the raster cell.
+
+The GDAL writer creates rasters using the data in the 'Z' dimension of the 
+points.  In a :ref:`pipeline_command` one can precede the GDAL writer with
+the ferry filter (:ref:`filters.ferry`) in order to copy data from some
+other dimension to the 'Z' dimension if desired.
+
```

Comment:
Is there a reason we can't use any given dimension and default to ``Z``? For example, Intensity rasters are a very common request. `filters.ferry` is a dirty hack that we don't want to institutionalize people into using IMO.